tests/page/locator-misc-2.spec.ts

```diff
@@ -69,8 +69,7 @@ it('should scroll zero-sized element into view', async ({ page, isAndroid }) =>
   `);
   expect(await page.locator('#lazyload').boundingBox()).toEqual({ x: 0, y: 2020, width: 1280, height: 0 });
   await page.locator('#lazyload').scrollIntoViewIfNeeded();
-  await page.evaluate(() => new Promise(requestAnimationFrame));
-  expect(await page.locator('#lazyload').textContent()).toBe('LAZY LOADED CONTENT');
+  await expect(page.locator('#lazyload')).toHaveText('LAZY LOADED CONTENT');
```

Comment:
What was the following:

```
await page.evaluate(() => new Promise(requestAnimationFrame));
```

trying to achieve before?

Reply:
I assume it was waiting for lazy-load to kick in!